app/src/main/res/values-hdpi/dimens.xml

```diff
@@ -1,22 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <dimen name="small">4dp</dimen>
+    <dimen name="medium">8dp</dimen>
+    <dimen name="large">12dp</dimen>
+    <dimen name="xl">16dp</dimen>
+    <dimen name="xxl">32dp</dimen>
```

Comment:
I think this dimensions could be more specific, so I would suggest renaming them for something like:
`margin_small, margin_medium, margin_large ...
`
or
`spacing_smll, spacing_medium, spacing_large ...
`

So it will be easier to call them when you are creating a layout, since they have same prefix. Also, you'll be abe to create specific text sizes in the same way:
`text_small = 8sp, text_medium = 12sp ...
`
or
`font_small, font_medium ...`

Reply:
This kind of naming will be really helpful. I shall use this convention during refactoring. As of now, these dimens are used for both marding and padding.